tangram_bundler/__init__.py

```diff
@@ -1,100 +1,235 @@
 #!/usr/bin/env python
 
 import os, sys, glob, yaml, zipfile
+from urlparse import urljoin
 
 # Append uniform textures
 # - uniforms could reference a texture already loaded from textures: {} or could explicitly define a texture file
-def addUniformTextureDependency(file_list, basePath, yaml_file, styleName, uniformName):
+def addUniformTextureDependency(fileList, basePath, rootNode, styleName, uniformName):
     referenceTexturePath = ""
     explicitUniformTexturePath = ""
-    key = yaml_file['styles'][styleName]['shaders']['uniforms'][uniformName]
+    key = rootNode['styles'][styleName]['shaders']['uniforms'][uniformName]
 
     if isinstance(key, basestring):
-        if ('textures' in yaml_file) and (key in yaml_file['textures']):
-            referenceTexturePath = basePath + '/' + yaml_file['textures'][ yaml_file['styles'][styleName]['shaders']['uniforms'][uniformName] ]['url']
-        explicitUniformTexturePath = basePath + '/' + yaml_file['styles'][styleName]['shaders']['uniforms'][uniformName]
+        if ('textures' in rootNode) and (key in rootNode['textures']):
```

Comment:
We also support arrays of textures in uniforms, e.g.

```
shaders:
  uniforms:
    textures: [textureA, textureB]
```

Reply:
Correct, will fix this. Thanks

Reply:
Addressed: 7a4cb9e

Found another bug with uniform texture sequence during import path resolution. 7a4cb9e also fixes this.